packages/netdog/netdog.spec

```diff
@@ -0,0 +1,96 @@
+%global _cross_first_party 1
+%undefine _debugsource_packages
+
+Name: %{_cross_os}netdog
+Version: 0.1.1
+Release: 0%{?dist}
+Summary: Bottlerocket network configuration helper
+License: Apache-2.0 OR MIT
+URL: https://github.com/bottlerocket-os/bottlerocket
+
+Source2: run-netdog.mount
+Source3: write-network-status.service
+Source4: netdog-tmpfiles.conf
+Source5: disable-udp-offload.service
+Source6: 00-resolved.conf
+Source7: generate-network-config.service
+
+BuildRequires: %{_cross_os}glibc-devel
+Requires: %{_cross_os}netdog
+%description
+%{summary}.
+
+%package -n %{_cross_os}netdog-systemd-networkd
+Summary: Bottlerocket network configuration helper
+Provides: %{_cross_os}netdog = 2:
+Requires: %{_cross_os}systemd-networkd
+Requires: %{_cross_os}systemd-resolved
+%description -n %{_cross_os}netdog-systemd-networkd
+%{summary}.
+
+%package -n %{_cross_os}netdog-wicked
+Summary: Bottlerocket network configuration helper
+Provides: %{_cross_os}netdog = 1:
+Requires: %{_cross_os}wicked
+%description -n %{_cross_os}netdog-wicked
+%{summary}.
+
+%prep
+%setup -T -c
+%cargo_prep
+
+%build
+mkdir bin
```

Comment:
nit: is this directory used anywhere?

Reply:
For the netdog binary itself i believe, needs to be created before hand